mne/viz/tests/test_evoked.py

```diff
@@ -385,6 +385,11 @@ def test_plot_compare_evokeds():
     plot_compare_evokeds(evoked_dict, cmap=('cmap title', 'inferno'),
                          linestyles=['-', ':', '--'])
     plt.close('all')
+    # test combine
+    match = 'must be an instance of None, callable, or str'
```

Comment:
Can the test check for the item_name that you set to combine too?

Reply:
@matteoanelli and I don't understand what you mean by that @rob-luke - can you clarify, please? :-)

Reply:
I think the idea is that the `match` string should be checking for `'combine'` as well, e.g.;
```suggestion
    match = 'combine must be an instance of None, callable, or str'
```